spec/system/jobseekers/jobseekers_can_give_account_feedback_spec.rb

```diff
@@ -33,4 +33,26 @@
     expect(current_path).to eq(jobseekers_account_path)
     expect(page).to have_content(I18n.t("jobseekers.account_feedbacks.create.success"))
   end
+
+  describe "email opt-out" do
+    it "updates the opt out field and adds a feedback record", :js do
+      click_on I18n.t("jobseekers.accounts.show.email_preferences.link")
+      choose I18n.t("helpers.label.jobseekers_email_preferences_form.email_opt_out_options.true")
+      click_button I18n.t("buttons.save_changes")
+      within ".govuk-list.govuk-error-summary__list" do
+        expect(all("li a").map(&:text)).to eq(["Select your reason for opting out"])
+      end
+      choose "Other"
+      fill_in "Tell us more (optional)", with: "Shant"
+      expect { click_save_and_wait_for_banner }.to change {
+        jobseeker.feedbacks.count
+      }.by(1)
+      expect(page).to have_content(I18n.t("jobseekers.email_preferences.update.success"))
+    end
+  end
+
+  def click_save_and_wait_for_banner
+    click_button I18n.t("buttons.save_changes")
+    find(".govuk-notification-banner")
+  end
```

Comment:
I think adding a couple of unit tests to jobseeker spec could be worthwhile, especially as we only test one of the validations in this system test.

Reply:
one of the validations was incorrect anyway.